src/dmd/cli.d

```diff
@@ -314,7 +314,42 @@ dmd -cov -unittest myprog.d
             "same as -i=-std,-core,-etc,-object"
         ),
         Option("i=[-]<pattern>,[-]<pattern>,...",
-            "include/exclude imported modules whose name matches one of <pattern>"
+            "include/exclude imported modules whose name matches one of <pattern>",
+            `this option enables "include import" mode, where the compiler will
+             include imported modules in the compilation, as if they were given
+             on the command line.
+
+             Determining which modules to include is based on a set of "module patterns".
+             These patterns are optionally included after the "-i=" option as a comma
+             separated list.  A pattern matches any module whose fully qualified name
+             starts with the pattern. The pattern "foo.bar" matches all modules in the
+             "foo.bar" package, such as "foo.bar" itself, "foo.bar.baz" or even
+             "foo.bar.package".  Note that matching is based on the full component name so
+             "foo.bar" would not match a module named "foo.barx".
+
+             The presence of the "-" prefix in a module pattern indicates that matching-modules
+             should be excluded from compilation.  Patterns without this indicate that
+             matching-modules should be included in compilation.
+
+             Note that along with the module patterns given by the user, there is a standard
```

Comment:
The style of addressing the user as "the user" is completely new and doesn't appear anywhere at https://dlang.org/dmd.html

I think sth. like this would fit better with the existing style:

> The standard set of exclusionary module patterns is "-std,-core,-etc,-object" and can be overridden with i.e. -i=std,core,etc,object).